### ISSUE TITLE
WES SVs default to "all callers" - MIP tag interaction issue?

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#3206]
All randomly drawn WES SV vars look like they were called by all callers. I would assume they are rather not tagged properly, and this is a default?

![Screenshot 2022-03-03 at 16 48 49](https://user-images.githubusercontent.com/758570/156600479-9533a8c6-3f4b-4931-8b52-f2250c951379.png)

**Comment by henrikstranneheim** [comment]
I think MIP only uses Manta for exome SV calling. The rest of the callers are turned off automatically

**Comment by dnil** [comment]
Yeah, that makes a lot of sense; will have a look at what it looks like caller merge status wise!

**Comment by dnil** [comment]
It appears very similar to https://github.com/Clinical-Genomics/MIP/issues/1934 - should be fixable in MIP with the same tactic as in https://github.com/Clinical-Genomics/MIP/pull/1936.

But, will see if we can't make the default in Scout a little more sensible without breaking anything.

**Comment by dnil** [comment]
I'm afraid that won't help in our case. The MIP10 WES calls are decorated with `set=Intersection` even though there is just one caller enabled. Then it becomes hard to separate from a case where multiple different callers are correctly intersected. I'll forward this to MIP instead since there are good solutions for this.

**Comment by dnil** [comment]
![Screenshot 2022-03-04 at 14 54 27](https://user-images.githubusercontent.com/758570/156776964-ef6e8c80-179a-42e3-817c-19685cbc07bb.png)
